src/app/store/machine/utils/common.ts

```diff
@@ -71,3 +75,19 @@ export const getHasSyncFailed = (machine?: Machine | null): boolean => {
   }
   return machine.is_sync_healthy === false;
 };
+
+/**
+ * Map the table sort direction to the value to send to the fetch request.
+ */
+export const mapSortDirection = (
+  sortDirection: ValueOf<typeof SortDirection>
+): FetchSortDirection | null => {
+  switch (sortDirection) {
+    case SortDirection.ASCENDING:
+      return FetchSortDirection.Ascending;
+    case SortDirection.DESCENDING:
+      return FetchSortDirection.Descending;
+    default:
+      return null;
+  }
+};
```

Comment:
This layer of abstraction could be avoided - SortDirection and FetchSortDirection values are exactly the same with the exception of `null`/`none`. I don't believe we will be adding another sort direction anytime soon and might as well use the same values on the front end and back-end.

Reply:
It's even worse because it's based on a type in react-components: https://github.com/canonical/react-components/blob/fd1b4e25ea97a88f205b4d8f2b0f03d8f6301302/src/types/index.ts#L19 but we also have our own version: https://github.com/canonical/maas-ui/blob/main/src/app/base/types.ts#L17.

I can't remember if/why MainTable needed the "none" option but I've added an issue to investigate to see if we can clean up these types: https://github.com/canonical/app-tribe/issues/1321 (until then we'll need this mapping so that we're not sending "none" to the backend).